documentation/JSON/V2.0.0_Hospital_price_transparency_schema.json

```diff
@@ -1,330 +1,473 @@
-                    },
-                    "minItems": 1
-                },
-                "standard_charges": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/standard_charges"
-                    },
-                    "minItems": 1
-                }
-            },
-            "required": [
-                "description",
-                "code_information",
-                "standard_charges"
-            ]
+        "setting": {
+          "enum": [
+            "inpatient",
+            "outpatient",
+            "both"
+          ],
+          "type": "string"
         },
-
         "payers_information": {
-            "type": "object",
-            "properties": {
-                "payer_name": {
-                    "type": "string",
-                    "minLength": 1
-                },
-                "plan_name": {
-                    "type": "string",
-                    "minLength": 1
-                },
-                "additional_payer_notes": {
-                    "type": "string"
-                },
-                "standard_charge_dollar": {
-                    "type": "number",
-                    "exclusiveMinimum": 0
-                },
-                "standard_charge_algorithm": {
-                    "type": "string"
-                },
-                "standard_charge_percentage": {
-                    "type": "number",
-                    "exclusiveMinimum": 0
-                },
-                "estimated_amount": {
-                    "type": "number",
-                    "exclusiveMinimum": 0
-                },                  
-                "methodology": {
-                    "enum": [
-                        "case rate",
-                        "fee schedule",
-                        "percent of total billed charges",
-                        "per diem",
-                        "other"
-                    ],
-                    "type": "string"
-                }
-            },
-            "required": [
-                "payer_name",
-                "plan_name",
-                "methodology"
-            ]
-          }
-    },
-    "type": "object",
-    "properties": {
-        "hospital_name": {
-            "type": "string",
-            "minLength": 1
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/payers_information"
+          },
+          "minItems": 1
         },
-        "hospital_address": {
+        "additional_generic_notes": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "setting"
+      ],
+      "anyOf": [
+        {
+          "type": "object",
+          "required": [
+            "gross_charge"
+          ]
+        },
+        {
+          "type": "object",
+          "required": [
+            "discounted_cash"
+          ]
+        },
+        {
+          "type": "object",
```

Comment:
This object needs to specify that `payers_information` is required. Otherwise, this would be a valid element in the `standard_charges` list, even though I think it violates conditional 2:
```json
{
  "minimum": 20000,
  "maximum": 20000,
  "setting": "inpatient"
}
```